src/routes/docs/products/databases/queries/+page.markdoc

```diff
@@ -418,3 +418,86 @@ query {
 }
 ```
 {% /multicode %}
+
+## Complex Queries
+You can create complex queries by combining AND and OR operations. For example, to find items in either the books or magazines category that are priced between $10 and $100.
+
+{% multicode %}
+```client-web
+const results = await databases.listDocuments(
+    '<DATABASE_ID>',
+    '<COLLECTION_ID>',
+    [
+        Query.or([
+            Query.equal('category', ['books', 'magazines'])
+        ]),
```

Comment:
This isn't quite realistic either because there's only 1 item in this or array.

How about book under 20 OR magazine under 10?

Reply:
@stnguyen90 How's the update?